src/components/form.tsx

```diff
@@ -382,7 +389,21 @@ const ModalUnitTitle = ({ unit, floor, postal, name, type }: TitleProps) => {
   }
   return (
     <Modal.Header>
-      <Modal.Title>{titleString}</Modal.Title>
+      <Modal.Title>
+        {titleString}
+        {unitPostal !== "undefined" && (
+          <span>
+            {", "}
+            <a
```

Comment:
Implement this as a button at the modal footer. Together w the other buttons.
Hyperlink is not very friendly to the non-techies.

Reply:
Changed to button

Reply:
@Fauxicing It is due to these unresolved conversations.